static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js

```diff
@@ -21,25 +21,17 @@ const IpsosTagging = function () {
     loadScript(ipsosSource, { id: 'ipsos', async: true, type: 'text/javascript' });
 };
 
-// Need to change to correct consent vendor
 export const init = (): Promise<void> => {
 
-    // Initial testing only
-    if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3")
-    {
-        IpsosTagging();
-    }
-
-    /*
-
-    onConsentChange(state => {
-        console.log(getConsentFor('a9', state));
-        if (getConsentFor('a9', state)) {
-            IpsosTagging();
-        }
-    });
-
-     */
+        onConsentChange(state => {
+            console.log(getConsentFor('ipsos', state));
+            // Initial testing only
+            if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3") {
+                if (getConsentFor('ipsos', state)) {
+                    IpsosTagging();
```

Comment:
I think the convention would be to lowercase the first character of the function name. Maybe it could be called `loadIpsosScript` as that is what it does.